### DISCUSSION TITLE
Liquid TryIt Editor module

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#16338] [Show and tell]
Hi,
I have developed a **Liquid TryIt Editor module** for [TOKYO website](https://showorchard.com/gallery/music/arts/tokyo-dance-music). It is a module that allows you to **try and learn Liquid language**.
As you can you see on the screenshot below, you can try any kind of Liquid script. This module also helped me a lot for learning Liquid and improve my development. This feature is permissible from the dashboard only to Administrator role.

<img width="1719" alt="Screenshot 2024-06-18 at 06 22 25" src="https://github.com/OrchardCMS/OrchardCore/assets/61869142/5aa7b8c1-a97a-4fe1-9152-db606fc167bf">

Therefore, if you like to include this module on next [OrchardCore releases](https://github.com/OrchardCMS/OrchardCore), I could open a pull request.

**Comment by hishamco** [comment]
I used Liquid Playground while I was working on Fluid, I think it's good for a community module. Let us hear the team feedback, or feel free to add it to Orchard Core Contrib (OCC) https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules

**Comment by ns8482e** [comment]
Isn't it the same as preview?

**Reply by DotCat1985** [reply]
It's the same

**Comment by DotCat1985** [comment]
Sorry, I don't understand. OrchardCoreContrib. Liquid contains only one filter.
Would you like my module to be integrated in OC solution? I'm expecting any team feedback.

**Comment by Piedone** [comment]
Awesome! Would you like it demo it on the next [Tuesday meeting](https://docs.orchardcore.net/en/latest/resources/meeting/)?

**Reply by DotCat1985** [reply]
I can demo it on July 11th.

**Comment by sebastienros** [comment]
I would instead document the ways you can do it online:
https://liquidjs.com/playground.html
https://shopify.dev/docs/api/liquid#liquid_basics
